protected/humhub/modules/file/models/File.php

```diff
@@ -8,56 +8,74 @@
 
 namespace humhub\modules\file\models;
 
+use humhub\components\ActiveRecord;
 use humhub\components\behaviors\GUID;
 use humhub\components\behaviors\PolymorphicRelation;
-use humhub\components\ActiveRecord;
+use humhub\libs\StdClass;
 use humhub\modules\content\components\ContentActiveRecord;
 use humhub\modules\content\components\ContentAddonActiveRecord;
+use humhub\modules\file\components\StorageManager;
+use humhub\modules\file\components\StorageManagerInterface;
 use humhub\modules\user\models\User;
+use Throwable;
 use Yii;
 use yii\base\InvalidArgumentException;
+use yii\base\InvalidConfigException;
+use yii\db\ActiveQuery;
 use yii\db\Exception;
+use yii\db\IntegrityException;
+use yii\db\StaleObjectException;
 use yii\helpers\Url;
 use yii\web\UploadedFile;
 
 /**
  * This is the model class for table "file".
  *
- * The followings are the available columns in table 'file':
+ * The following are the available columns in table 'file':
+ *
  * @property integer $id
  * @property string $guid
  * @property string $file_name
  * @property string $title
  * @property string $mime_type
  * @property string $size
- * @property string $object_model
- * @property integer $object_id
- * @property integer $content_id
- * @property string $created_at
- * @property integer $created_by
- * @property string $updated_at
- * @property integer $updated_by
- * @property integer $show_in_stream
- * @property string $hash_sha1
+ * @property string|null $object_model
+ * @property integer|null $object_id
+ * @property integer|null $content_id
+ * @property string|null $created_at
+ * @property integer|null $created_by
+ * @property string|null $updated_at
+ * @property integer|null $updated_by
+ * @property integer|null $show_in_stream
+ * @property string|null $hash_sha1
  *
- * @property \humhub\modules\user\models\User $createdBy
- * @property \humhub\modules\file\components\StorageManager $store
+ * @property StorageManager $store
  * @property FileHistory[] $historyFiles
```

Comment:
We should also add the `metadata` property here.

Reply:
Absolutely. Got lost in all the cherry-picking, rebasing, and merging! 🤣